src/js/modalRating.js

```diff
@@ -79,54 +94,57 @@ document.addEventListener('DOMContentLoaded', function() {
         });
       };
 
-      const isValidEmail = (email) => {
+      const isValidEmail = email => {
         const emailRegex = /^\w+@[a-zA-Z_]+?\.[a-zA-Z]{2,3}$/;
         return emailRegex.test(email);
       };
 
-      const handleSubmit = async (event) => {
-        event.preventDefault()
+      const handleSubmit = async event => {
+        event.preventDefault();
         const starId = selectedStar?.id;
         const starRating = parseInt(starId?.replace('star', ''), 10);
-      
+
         if (!starRating || isNaN(starRating)) {
-          iziToast.show({ message: 'Please add your rating' });
+          showMessage('info', 'Please add your rating');
```

Comment:
is title required here?

Reply:
title is used for function to determine type of message, but won't be shown